README-SUBMODULES.md

```diff
@@ -0,0 +1,61 @@
+# How to convert Mercurial Repositories with subrepos
+
+## Introduction
+
+First a mapping file needs to be created that maps between the subrepo folder
+and the corresponding git submodule. The format for this file is:
+
+"<mercurial subrepo folder>"="<git submodule name>"
+"<mercurial subrepo folder2>"="<git submodule name2>"
+...
+
+## Example
```

Comment:
I meant that you should start the introduction by telling the user that the subrepos must be converted first in order for the conversion of the super repo to know how hg commits map to git commits in the sub repos. Then explain how the mapping file (and its syntax) is used to find the git repo for a subrepo directory.

An advanced (or repeat) user will then have enough information to to the conversion, for everyone else we have the example.